src/Vatras/Data/Prop/Properties.agda

```diff
@@ -0,0 +1,134 @@
+module Vatras.Data.Prop.Properties {F : Set} where
+
+open import Data.Bool as Bool using ()
+open import Data.Bool.Properties using (∧-comm; ∧-zeroˡ; ∧-zeroʳ)
+open import Data.Empty using (⊥)
+open import Data.Product as Product using (Σ; _×_; ∃-syntax; _,_)
+open import Data.Sum as Sum using (_⊎_) renaming (inj₁ to left; inj₂ to right)
+
+open import Relation.Nullary.Negation renaming (¬_ to never)
```

Comment:
Here, the rename makes sense to avoid name clashes.
However, for me `never (Const p)` doesn't make a lost of sense. What about `negate (Const p)` or `Negation.¬ (Const p)`